src/02-common-principles.md

```diff
@@ -303,6 +303,30 @@ sub- prefix is not part of the subject label, but must be included in file names
 MUST be integers. Those labels MAY include zero padding, but this is NOT
 RECOMMENDED to maintain their uniqueness.
 
+## The contrast suffix
```

Comment:
I don't think the term `contrast` applies to all data types covered by BIDS (think about MEG, fieldmaps, physiological recordings etc.). I think this clarification fits better into anatomical MR section.

Reply:
The term contrast should apply equally well to any multidimensional array of data. Perhaps I should clarify that each image intrinsically has a contrast and that this does not imply the administration of a “contrast agent” (which is simply an experimental manipulation which increases the dynamic range of the contrast which the image has either way)? Is this what you were thinking of?

For fieldmaps the contrast is the b0 strength, for physiological recordings it's motion, or heart rate, etc. The only place where I see an inconsistency is with tabular data representation such as for the event files, which I attempted to clarify.

Reply:
I only listed some and there are many others. They are all described through the spec - you can look them up. I agree that it would've been nice to have a coherent definition of the meaning of suffix, but I don't think this is currently possible without breaking backward compatibility. At the end of the day, suffix is an “ad-hoc identifier which helps us select the files” - not necessarily by design, but we have to live with this now.

Reply:
I understand your point. Ok, so if there is no unifying feature about this field other than being a suffix, we can perhaps just rename it from “modality” to just that, “suffix”?

Reply:
Sure!

Reply:
so you would be ok with me synthetizing the discussion we had here about the intrinsic variability of the `_suffix` and its role primarily as a disambiguator for cursory users, but adding a paragraph on how for NIfTI files this is equivalent to the contrast, and how the contrast in this sense is separate from both the “contrast agent” and the “acquisition” (which both have their own fields). y?

Reply:
Go ahead - however, I am not convinced this is the most pressing issue to work on and I am worried that the description of `suffix` with its exceptions will be of limited use.

Working on a PR adding support for CBV data might be a better investment of your time and skills.

Reply:
I thought that was already merged?

Reply:
indeed it was - my bad!